key,default_value
userAgent,"Botocore/1.16.26 Python/2.7.18 Linux/5.4.0-2-amd64"
eventID,"677f78a0-aa4a-494d-8c95-01cbb8658cbb"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: events:ListEventSources on resource: arn:aws:events:us-east-1::event-source/*"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"251.105.254.1"
eventName,"ListEventSources"
eventSource,"events.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"b60ebdc7-0bc0-45d9-8903-c25fd5be2554"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-06-10T05:35:15Z"
